eclair-core/src/main/resources/reference.conf

```
@@ -90,8 +90,17 @@ eclair {
   max-block-processing-delay = 30 seconds // we add a random delay before processing blocks, capped at this value, to prevent herd effect
   max-tx-publish-retry-delay = 60 seconds // we add a random delay before retrying failed transaction publication
 
-  fee-base-msat = 1000
-  fee-proportional-millionths = 100 // fee charged per transferred satoshi in millionths of a satoshi (100 = 0.01%)
+  // Fees for public channels
+  fee-base-msat-public = 1000
+  fee-proportional-millionths-public = 200 // fee charged per transferred satoshi in millionths of a satoshi (200 = 0.02%)
+
+  // Fees for private channels
+  fee-base-msat-private = 1000
+  fee-proportional-millionths-private = 100
+
+  // Minimum fees for trampoline relays
+  min-trampoline-fee-base-msat = 1000
+  min-trampoline-fee-proportional-millionths = 100
```

I would go with a more hierarchical organization. We have been going in that direction with `router`, `peer-connection`, `db` and I'd like to move channel-related parameters to a `channel` section at some point.

Those fee params are kind of in-between `channel` and `relay` but since there are trampoline-related settings, it makes sense to put them under `relay` I think.

```suggestion
  relay.fees {
    // Fees for public channels
    public-channels {
      fee-base-msat = 1000
      fee-proportional-millionths = 200 // fee charged per transferred satoshi in millionths of a satoshi (200 = 0.02%)
    }
    // Fees for private channels
    private-channels {
      fee-base-msat = 1000
      fee-proportional-millionths = 100
    }
    // Minimum fees for trampoline relays
    min-trampoline {
      fee-base-msat = 1000
      fee-proportional-millionths = 100
    }
```

This structure makes it possible to factorize parsing too.